src/llmcompressor/utils/metric_logging.py

```diff
@@ -39,6 +46,36 @@ def get_GPU_memory_usage() -> List[Tuple]:
         return []
 
 
+def get_GPU_usage_amd() -> List[Tuple[float, float]]:
```

Comment:
Quick comments within this method would be helpful

Reply:
Done ! Thanks :)